### ISSUE TITLE
Function Node Editor: Wot No 'Paste' option in Right-Click Menu?

### ISSUE THREAD

**Issue by 3ative** [node-red/node-red-nodes#946]
### Which node are you reporting an issue on?
Function
### What are the steps to reproduce?
* Select some characters
* Right-Click Choose Copy or Cut
* Right-Click elsewhere
### What happens?
Menu pops up but, there's no 'Paste' listed

![Capture](https://user-images.githubusercontent.com/51385971/187841041-1475e09c-3616-43f1-b891-f9df3e56f377.JPG)
### What do you expect to happen?
Right-Click to open menu, and there's a 'Paste' option

### Please tell us about your environment:

- [x] Node-RED version: 3.0.2: Maintenance Release
- [ ] node.js version:
- [ ] npm version:
- [x] Platform/OS: HASSIO / Add-on
- [x] Browser: Chrome Version 104.0.5112.102

**Comment by knolleary** [comment]
Are you accessing the editor using http or https?

Due to browser security restrictions, paste is only available when using either localhost, or an https URL.

See https://github.com/microsoft/monaco-editor/issues/3003 for example.

**Comment by 3ative** [comment]
Thanks for the reply @knolleary 

I'm running the NR Hadd-on within Home Assistant so it'sL "http://hassio.local:8123/a0d7b954_nodered/dashboard"
Not sure if this qualifies as a 'local.host' but it's defo not HTTPS.

I mainly use Ctrl-V anyway 😁 I only raised the issue thinking it may be a simple oversight. The issue link you provided explains everything I need to know - Thanks again. 

Closing this one now. Cheers.
